files-community/Files
# Feature: Added a List View layout

**Resolved / Related Issues**
   Closes #2509...

**Validation**
How did you test these changes?
   - Removed border from selected items in the Grid View layouts
   1. Switch to List View using the toolbar flyout, context menu, `ctrl` + `shift` + `2` shortcut
   2. Switch from the Details View to List View using `ctrl` + `+` and from the Grid View using `ctrl` + `-`
   3. Set the default view to List and confirm it works as expected
   4. Enable compact spacing and confirm it works as expected
   5. Rename items in the List View and confirm it works as expected


**Screenshots (optional)**
![image](https://github.com/files-community/Files/assets/39923744/745c5615-59a0-4660-b2ac-49eb25334574)
